### PR TITLE
bt-461: Create characteristics component in admin user verification

### LINKED ISSUE #461: Create characteristics component in admin user verification
**User story:**
As an admin, I want to have a characteristics tab on the verification screen, so that I can choose badges and add them to the candidate's profile

**Acceptance criteria:**
- Feedback block should have feedback from each week, Feedback from BSA coach project, HR interview results
- When the admin clicks on Feedback - the comment should be displayed from the right side
- Admin should be able to choose Personality type, Soft skills of a talent
- Personality type dropdown field should be multichoice and have options: Doer / Connector / Thinker
- Soft skills dropdown field should be multichoice and have options: Communicative, Collaboration, Problem-solving, Leadership, Creative, Proactive
- Soft skills and Personality type should have placeholder “Option”

Image

### PR DESCRIPTION
**Pull request by Monyster**
Closes #461 

image

### CODE REVIEW

File: frontend/src/bundles/common/components/multi-read/styles.module.scss [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    flex: 1 1 auto;
+    padding: 13px 17px;
+    background: var(--white-lilac);
+    border: 1px solid #f6f7f9;
+    border-radius: 4px;
+}
+
+.text {
+    color: #7c8798;
+}
```
Comment (nika-sirak) [inline frontend/src/bundles/common/components/multi-read/styles.module.scss:50]: --slate-gray: #7c8798;

File: frontend/src/bundles/common/components/multi-read/styles.module.scss [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    background-color: var(--st-tropaz);
+    border-radius: 50%;
+    content: " ";
+}
+
+.textWrapper {
+    flex: 1 1 auto;
+    padding: 13px 17px;
+    background: var(--white-lilac);
+    border: 1px solid #f6f7f9;
```
Comment (nika-sirak) [inline frontend/src/bundles/common/components/multi-read/styles.module.scss]: maybe better add to css variables

File: frontend/src/bundles/admin/components/characteristics/characteristics.tsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+                    Feedback
+                </Typography>
+                <Grid
+                    container
+                    item
+                    className={styles.body}
+                    style={{
+                        borderRadius: '8px',
+                        border: '1px solid #DCE5FF',
+                        padding: '20px',
```
Comment (nika-sirak) [inline frontend/src/bundles/admin/components/characteristics/characteristics.tsx]: Can you put those styles in scss file?

File: frontend/src/bundles/admin/components/characteristics/characteristics.tsx [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+    MultiRead,
+    Select,
+    Typography,
+} from '~/bundles/common/components/components.js';
+import { getValidClassNames } from '~/bundles/common/helpers/helpers.js';
+import { useAppForm } from '~/bundles/common/hooks/hooks.js';
+
+import styles from './styles.module.scss';
+
+// Mock data
```
Comment (nikita-remeslov) [inline frontend/src/bundles/admin/components/characteristics/characteristics.tsx]: add todo comment, to remove it

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]

File: frontend/src/bundles/admin/components/characteristics/characteristics.tsx (hunk 4 above)
Comment (Monyster) [reply frontend/src/bundles/admin/components/characteristics/characteristics.tsx]: changed to mock-data/mock-personality-type-options.ts

**Review by nikita-remeslov — APPROVED** [review]

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**